pkg/metrics/metrics.jsx

```diff
@@ -493,7 +526,19 @@ class CurrentMetrics extends React.Component {
         return (
             <Gallery className="current-metrics" hasGutter>
                 <Card id="current-metrics-card-cpu">
-                    <CardTitle>{ _("CPU") }</CardTitle>
+                    <CardHeader>
+                        { !isNaN(this.cpuTemperature) &&
+                        <CardActions>
+                            <HelperText>
+                                <HelperTextItem hasIcon variant={
+                                    this.cpuTemperature < 80 ? "warning" : (this.cpuTemperature < 95 ? "warning" : "error")
+                                }>
+                                    { cockpit.format("$0 °C", this.cpuTemperature) }
+                                </HelperTextItem>
+                            </HelperText>
```

Comment:
This should not use a HelperText as it is not part of a form. Most likely you want to put the Icon and text in a Flex, and then `align-items: baseline` should work. 

baseline now does not work as CSS aligns the outer html element and not the text within the `<CardActions>`